detection_rules/main.py

```diff
@@ -113,8 +113,8 @@ def name_to_filename(name):
 
 
 @root.command('toml-lint')
-@click.option('--rule-file', '-f', type=click.Path(), multiple=True,
-              help='Optionally specify rule files')
+@click.option('--rule-file', '-f', multiple=True, type=click.Path('r'),
+              help='Optionally specify a specific rule file only')
```

Comment:
this doesn't make sense because we have `--multiple=True`

Reply:
ok so this syntax works for multiple, I've updated the help prompt
```
python -m detection_rules toml-lint -f "C:\dev\Elastic repos\fork-detection-rules\detection-rules\rules\aws\defense_evasion_cloudtrail_logging_suspended.toml" -f "C:\dev\Elastic repos\fork-detection-rules\detection-rules\rules\aws\defense_evasion_configuration_recorder_stopped.toml"
```